README.md

```diff
@@ -81,6 +81,14 @@ java -jar composer-latest-version.jar \
 --verbose-output false
 ```
 
+### Download
+
+Composer [published to jcenter](https://jcenter.bintray.com/com/gojuno/composer) and follows [Semantic Versioning](http://semver.org/).
```

Comment:
`Composer is available at jcenter`. Versioning note is kind of useless in most cases.